components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -151,231 +151,224 @@ const ContributorHighlightCard = ({ title, desc, prLink, user, id }: Contributor
+                  >
+                    <FiLinkedin size={22} />
+                    <span>Share to Linkedin</span>
+                  </a>
+                </DropdownMenuItem>
+                <DropdownMenuItem
+                  onClick={() => handleCopyToClipboard(`${host}/feed/${id}`)}
+                  className="rounded-md flex gap-2.5 py-1 items-center pl-3 pr-7"
+                >
+                  <BsLink45Deg size={22} />
+                  <span>Copy link</span>
+                </DropdownMenuItem>
+                <DropdownMenuItem className="rounded-md hidden">
+                  <div className="flex gap-2.5 py-1  items-center pl-3 pr-7">
+                    <FaUserPlus size={22} />
+                    <span>Follow user</span>
+                  </div>
+                </DropdownMenuItem>
+                {loggedInUser && (
                   <DropdownMenuItem
                     className={`rounded-md ${
-                      loggedInUser && loggedInUser.user_metadata.user_name === user && "hidden"
+                      loggedInUser && loggedInUser.user_metadata.user_name !== user && "hidden"
                     }`}
                   >
-                    <a
-                      href={`mailto:hello@opensauced.pizza?subject=${reportSubject}`}
-                      className="flex gap-2.5 py-1  items-center pl-3 pr-7"
-                    >
-                      <GrFlag size={22} />
-                      <span>Report content</span>
-                    </a>
+                    <button onClick={() => setOpenEdit(true)} className="flex w-full cursor-default gap-2.5 py-1  items-center pl-3 pr-7">
+                      <FiEdit size={22} />
+                      <span>Edit</span>
+                    </button>
                   </DropdownMenuItem>
-                </DropdownMenuContent>
-              </DropdownMenu>
-            </div>
-          </div>
+                )}
 
-          {/* Highlight body section */}
-          <div className="w-full ">
-            <p className="text-light-slate-11 break-words text-sm lg:text-base font-normal">{desc}</p>
+                <DropdownMenuItem
+                  className={`rounded-md ${
+                    loggedInUser && loggedInUser.user_metadata.user_name === user && "hidden"
+                  }`}
+                >
+                  <a
+                    href={`mailto:hello@opensauced.pizza?subject=${reportSubject}`}
+                    className="flex gap-2.5 py-1  items-center pl-3 pr-7"
+                  >
+                    <GrFlag size={22} />
+                    <span>Report content</span>
+                  </a>
+                </DropdownMenuItem>
+              </DropdownMenuContent>
+            </DropdownMenu>
           </div>
-          {/* Highlight Link section */}
+        </div>
 
-          <div>
-            <a href={prLink} className="underline text-sauced-orange cursor-pointer">
```

Comment:
@bdougie, should this be changed to a regular `p` element or given the `target="_blank"` attribute to open the link in a new tab?

Reply:
Adding `target=_blank` would be good